{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name}}/validation/notebooks/ModelValidation.py

```diff
@@ -116,59 +131,40 @@ def get_targets_from_recipe():
 
 # COMMAND ----------
 
-##################################################################################
-# TODO : Please fill in this cell with proper values for enable_baseline_comparison,
-#        data, targets, model_type, custom_metrics, validation_thresholds
-#        evaluator_config
-##################################################################################
+from model_validation_input import (
+    enable_baseline_comparison,
+    get_prod_workspace_validation_input,
+    get_staging_workspace_validation_input,
+    get_model_type,
+    get_validation_thresholds,
+    get_custom_metrics,
+    get_evaluator_config,
+)
 
-# Whether to load the current registered "Production" stage model as baseline. A version with "Production" stage must
-# exist for the model.
-# Baseline model is a requirement for relative change and absolute change validation rules.
-# TODO(optional) : enable_baseline_comparison
-enable_baseline_comparison = False
+# take input
+enable_baseline_comparison = enable_baseline_comparison()
 
-# model validation data input for staging or prod workspace. A Pandas DataFrame or Spark DataFrame, containing evaluation features and labels.
-# Please refer to data parameter in mlflow.evaluate documentation https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate
 if env == "prod":
-    # TODO(required) : set data for prod workspace
-    data = None
+    data = get_prod_workspace_validation_input(spark)
```

Comment:
I have to pass the `spark` object.
When I used `from databricks.sdk.runtime import spark` in the input python file, it works fine when I manually run the notebook but throws error when I run the workflow. 
It's probably related to MLR version.